spec/core/v2/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -596,15 +596,33 @@ function sendPacket(
     // increment the sequence. Thus there are monotonically increasing sequences for packet flow for a given clientId
     nextSequenceSend[sourceChannelId]=sequence+1
     
-    // log that a packet can be safely sent
-    // Event Emission 
-    emitLogEntry("sendPacket", {
+    // Event Emission for send packet
+    emitEvents("send_packet", {
       sourceId: sourceChannelId, 
```

Comment:
Is there any particular reason why we use sourceId here rather than sourceChannelId? I don't care too much either way, but why not use the same names as the packet structure?